ENIGMAsystem/SHELL/Platforms/xlib/Linux/joystick.h

```diff
@@ -0,0 +1,44 @@
+/********************************************************************************\
+**                                                                              **
+**  Copyright (C) 2008 Josh Ventura                                             **
+**                                                                              **
+**  This file is a part of the ENIGMA Development Environment.                  **
+**                                                                              **
+**                                                                              **
+**  ENIGMA is free software: you can redistribute it and/or modify it under the **
+**  terms of the GNU General Public License as published by the Free Software   **
+**  Foundation, version 3 of the license or any later version.                  **
+**                                                                              **
+**  This application and its source code is distributed AS-IS, WITHOUT ANY      **
+**  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS   **
+**  FOR A PARTICULAR PURPOSE. See the GNU General Public License for more       **
+**  details.                                                                    **
+**                                                                              **
+**  You should have recieved a copy of the GNU General Public License along     **
+**  with this code. If not, see <http://www.gnu.org/licenses/>                  **
+**                                                                              **
+**  ENIGMA is an environment designed to create games and other programs with a **
+**  high-level, fully compilable language. Developers of ENIGMA or anything     **
+**  associated with ENIGMA are in no way responsible for its users or           **
+**  applications created by its users, or damages caused by the environment     **
+**  or programs made in the environment.                                        **
+**                                                                              **
+\********************************************************************************/
+
+#include "../../General/PFjoystick.h"
+
+namespace enigma {
+  void init_joysticks();
```

Comment:
I can make this declared once if you want, but it doesn't belong in the general header unless Windows has it too. Because I am in a good mood I'll check and see if that's the case and will move it if it is. But really this problem existed before this pr between linux and windows, so it's not my job to do additional housekeeping in a pr that wasn't meant to address that.